Data-Structures/HashTable/HashTable.js

```diff
@@ -0,0 +1,105 @@
+export default class HashTable {
+  constructor(limit = 100) {
```

Comment:
100 seems very arbitrary. I'd force the user to choose for a fixed-size hash table. (I'd also make it clear that this is a fixed-size hash table (with dynamically grown buckets) as opposed to a dynamic hash table. In particular, a fixed-size hash table can't guarantee expected O(1) operations.)

`limit` is also a slightly misleading name: This is not a hard limit, just a hash table "size". If you insert more elements, you will just get worse performance as your buckets fill up.